### ISSUE TITLE
Feature request : ability to manually reorganize production buildings order

### ISSUE THREAD

**Issue by Blackhole1504** [Jacudibu/PRUNner#26]
First of all, thanks for the new update ! It's amazing and more intuitive :+1: 

It's a minor inconvenience but will it be possible to manually reorganize the buildings  ?

**Comment by Jacudibu** [comment]
Yeah, shouldn't be too hard to add some way to drag&drop the things or just some small buttons to manually reorder them
